ci-operator/step-registry/openshift/upgrade/aws/heterogeneous/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- wking
+- jluhrsen
+- deads2k
```

Comment:
I am not sure about a good set of reviewers here, this is a copy paste of others' upgrade workflow OWNERS file.

Reply:
It is probably worth including someone from multi-arch in here, so they can approve new `upgrade-aws-heterogeneous-*` steps/chains/workflows if we need more in the future.  @deepsm007 , would that be you?  Or can you recommend someone else?

I don't think we need to block on having someone from multi-arch though.  If we stick with the parent-dir approver set, `hack/dedup-owners.sh` makes it easy to set up a DRYer symlink.

Reply:
@deepsm007 and @Prashanth684 to own heterogeneous